buildSrc/src/main/java/com/osacky/flank/gradle/RunFlankTask.kt

```diff
@@ -0,0 +1,11 @@
+package com.osacky.flank.gradle
+
+import org.gradle.api.DefaultTask
+import javax.inject.Inject
+
+open class RunFlankTask @Inject constructor(private val extension: FlankGradleExtension) : DefaultTask() {
```

Comment:
@winterDroid In this pull request, I moved the run flank task to a separate class so we can add parameters to it in the future.
This class `dependsOn` the actual tasks which will download and run flank.
We'll have to figure out how to pass parameters from this task to its dependencies, but I think this is the right starting point. What do you think @winterDroid ?

Reply:
Yeah that's probably a good starting point 👍